docs/source/features/file-uploads.md

```diff
@@ -0,0 +1,139 @@
+---
+title: File Uploads
+description: Implementing File Uploads on Apollo Server
+---
+
+File uploads are a requirement for many applications. Apollo Server supports the [GraphQL multipart request spec](https://github.com/jaydenseric/graphql-multipart-request-spec) for uploading files as mutation arguments.
+
+## File upload with default options
+
+Apollo Server automatically adds the `Upload` scalar to the schema, so any existing declaration of `scalar Upload` in the schema should be removed.
+
+```js
+const { ApolloServer, gql } = require('apollo-server');
+
+const typeDefs = gql`
+  type File {
+    filename: String!
+    mimetype: String!
+    encoding: String!
+  }
+
+  type Query {
+    uploads: [File]
+  }
+
+  type Mutation {
+    singleUpload(file: Upload!): File!
+  }
+`;
+
+const resolvers = {
+  Query: {
+    files: () => {
+      // Return the record of files uploaded.
```

Comment:
This is a bit confusing to me where the record might come from. Adding a short snippet about where the record comes from: file system, db, or api call would be helpful.